Source/ParameterEncoder.swift

```diff
@@ -221,6 +224,45 @@ public final class URLEncodedFormEncoder {
         }
     }
 
+    /// Configures how `Date` parameters are encoded.
+    public enum DateEncoding {
+        /// Encodes the date from `Date` structure.
+        case deferredToDate
+        /// Encodes the date as a UNIX timestamp.
+        case secondsSince1970
+        /// Encodes the date as UNIX millisecond timestamp.
+        case millisecondsSince1970
+        /// Encodes the date as an ISO-8601-formatted string (in RFC 3339 format).
+        case iso8601
+        /// Encodes the date as a string formatted by the given formatter.
+        case formatted(DateFormatter)
+        /// Encodes the date as a custom string formatted by the given closure.
+        case custom((Date) -> String)
```

Comment:
I added `throws` attribute. But there's a problem with `@escaping`: there is an error in the Apple documentation. This closure cannot be declared as escaping because it is already stored and therefore escaped.